backend/src/loaders/territory/territory.js

```diff
@@ -0,0 +1,25 @@
+import each from 'lodash/each';
+import { lockdownSheetUrl } from '../../config.js';
+import { writeJSON } from '../../utils/file.js';
+
+export function parseData(territory) {
+  return {
+    contribute_url: `${lockdownSheetUrl}/edit#gid=${territory['googlesheet_tab_id']}`,
+  }
+}
+
+export async function getData() {
+  const territories = {};
+  const territoriesMeta = require('./meta.json');
+  each(territoriesMeta, (territory, isoCode) => {
```

Comment:
doesnt this do the same as Array.forEach?

Reply:
Actually it was an object, not array. Anyways I've replaced with native implementation, probably going to ditch lodash soon if no use case